### PR TITLE
Extend and clean up the top-bar

### PR DESCRIPTION
**Pull request by da-kami**
Add the reference price including tooltip & link to oracle outcome explorer.
Combine the funding event time and rate.
Add color to maker status.

Set min-widths so the text does not jump around due to resizing based on text.

looks: 

image

### CODE REVIEW

**Review by bonomat — APPROVED** [review]
Getting crowded in the bar :)

File: taker-frontend/src/components/NavBar.tsx [hunk 1]
```diff
@@ -77,29 +90,48 @@ export default function Nav({ walletInfo, connectedToMaker, fundingRate, nextFun
...
                             alignItems={"center"}
                         >
-                            <Heading size={"sm"}>{nextFundingEvent}</Heading>
+                            <Tooltip
+                                label={"The next time your CFDs will be extended and the funding fee will be collected based on the hourly rate."}
+                                hasArrow
+                            >
+                                <HStack minWidth={"260px"}>
+                                    <Heading size={"sm"}>{nextFundingEvent}</Heading>
+                                    <Text>{"@ a rate of"}</Text>
```
Comment (thomaseizinger) [inline taker-frontend/src/components/NavBar.tsx]: % implies a rate so we could drop this "a rate of" text if you wanted to save some more space?

**Review by thomaseizinger — APPROVED** [review]

**Comment by klochowicz** [comment]
I love the new bar!!! 💯

**Review by klochowicz — APPROVED** [review]

**Pull request merged**